src/components/Icon/BookMarkIcon.tsx

```diff
@@ -0,0 +1,32 @@
+interface BookMarkIconProps {
+  onChange?: () => void;
+}
```

Comment:
Icon should not have this prop, this should belong to parent element

Reply:
Fixed